persistence/cosmosdb/transactions_information_CosmosDB_[1.1,].partial.md

```diff
@@ -0,0 +1,91 @@
+A configuration API has been introduced as of version 1.1 and above. This configuration API provides the ability to specify the partition and Container used per message using message headers or the message contents.
```

Comment:
Maybe also mention something about the order of execution and when one matched all others will not be executed

Reply:
See also https://github.com/Particular/NServiceBus.Persistence.CosmosDB/blob/master/src/NServiceBus.Persistence.CosmosDB/Transaction/TransactionInformationConfiguration.ContainerInformation.cs#L16

Reply:
Done